bubble_sort/Python/bubbleSort.py

```diff
@@ -0,0 +1,24 @@
+#Bubble Sort in Python
+
+array = [] #declaration of array
```

Comment:
E262 inline comment should start with '# '

*Origin: PycodestyleBear (E262), Section: `all.autopep8`.*